### PR TITLE
GG-001

### PR DESCRIPTION
**Pull request by sMartz1**
Create HEADER component

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+
+import './component.css'
```
Comment (helkyar) [inline src/components/Header/Header.jsx:3]: Even thought "Header" is a widely used component. Shouldn't it be a better option to give it its own css?

**Review by helkyar — COMMENTED** [review]
All in all, LGTM!

File: src/components/Header/Header.jsx (hunk 1 above)
Comment (sMartz1) [reply src/components/Header/Header.jsx:3]: It currently has its own .header class

If you mean css file, the component.css is a file that only contains the css related to header.
Path src/components/Header/component.css

Comment (sMartz1) [reply src/components/Header/Header.jsx:3]: We will standardize the component folder at a later date

**Pull request merged**